deploy/kubernetes/csi-nfs-controller.yaml

```diff
@@ -62,7 +61,7 @@ spec:
               cpu: 10m
               memory: 20Mi
         - name: nfs
-          image: quay.io/k8scsi/nfsplugin:v2.0.0
+          image: gcr.io/k8s-staging-sig-storage/nfsplugin:amd64-linux-canary
```

Comment:
fyi, we disabled the cloud build job for nfs because it has never passed:
https://github.com/kubernetes/test-infra/pull/19937/files#diff-6572b562b74e036959cc413b3e4b7eb275f82e89218a1b1eb36a3dfb136348cdR39

https://k8s-testgrid.appspot.com/sig-storage-image-build#post-csi-driver-nfs-push-images

Can you help fix the job?

Reply:
@mayankshah1607 could you help? thanks.

Reply:
cc @songjiaxun

Reply:
filed an issue to track: https://github.com/kubernetes-csi/csi-driver-nfs/issues/94, need to fix it ASAP